packages/hop-node/src/watchers/classes/Bridge.ts

```diff
@@ -67,6 +69,8 @@ export default class Bridge extends ContractBase {
       throw new Error('bridge deployed block number is required')
     }
     this.bridgeDeployedBlockNumber = config.tokens[this.tokenSymbol]?.[this.chainSlug]?.bridgeDeployedBlockNumber
+    this.minAmount = config?.bondWithdrawals?.[this.chainSlug]?.[this.tokenSymbol]?.min ?? 0
```

Comment:
shouldn't these be converted to BigNumber types?

Reply:
Resolved with ae73d900762faa1f5898955d07d840716155bc9b